packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -124,9 +130,17 @@ $detailsList-item-focus-meta-text-color: $ms-color-neutralDark;
     .cell {
       color: $detailsList-item-selected-hover-meta-text-color;
 
+      @media screen and (-ms-high-contrast: active) {
```

Comment:
There wasn't a `high-contrast` mixin available outside of fabric-website so I added one to common for now. Seems like there is an open feature request in fabric-core to add these mixins, but for now they're in fabric react common.